packages/cadl-ranch-specs/http/azure/core/traits/mockapi.ts

```diff
@@ -33,35 +33,6 @@ Scenarios.Azure_Core_Traits_smokeTest = passOnSuccess({
       "x-ms-client-request-id": "86aede1f-96fa-4e7f-b1e1-bf8a947cb804",
     },
   },
-  handler: (req: MockRequest) => {
-    if (!("x-ms-client-request-id" in req.headers)) {
-      throw new ValidationError("Should submit header x-ms-client-request-id", "any uuid", undefined);
-    }
```

Comment:
Same, here the validation is only GUID. (the header is automatically generated by SDK, hence "any uuid")

Reply:
Added the handler back.